max-rps/max-rps.go

```diff
@@ -65,7 +65,7 @@ func (cfg Config) Run() {
 		return mismatch
 	}
 
-	grad := func(grad, x []float64) []float64 {
+	grad := func(grad, x []float64) {
```

Comment:
This was oddly the only API difference from the update and also the source of the issue. Even though `gonum` was only a minor version bump, it's still < 1.0 so it makes sense that it could be breaking.